### PR TITLE
Add Homepage

### PR DESCRIPTION
**Pull request by kaustubh-jsr**
Added homepage screen

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+.caraousel-arrow{
+    background-color: rgb(255, 255, 255);
+    box-shadow: rgb(0 0 0 / 16%) 0px 0px 12px 0px;
+    color: #333;
+    margin: 0 1rem;
+}
+
+.home-page-category-display{
+    display: flex;
```
Comment (amantulsyan35) [inline css/homepage.css]: instead of home-page can make it hompage

Comment (amantulsyan35) [reply css/homepage.css]: *homepage

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+.home-page-caraousel{
+    /* border: 1px solid black; */
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.caraousel-arrow{
+    background-color: rgb(255, 255, 255);
+    box-shadow: rgb(0 0 0 / 16%) 0px 0px 12px 0px;
```
Comment (swstica) [inline css/homepage.css]: It is better to use color variables instead of hex/rgb

File: index.html [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+                </div>
+                <div class="card category-cards__sample">
+                    <img src="https://www.bigbasket.com/media/customPage/b01eee88-e6bc-410e-993c-dedd012cf04b/bdcb2f5b-a1c4-4cd7-aab4-dd41e7a17532/0e1fa913-525a-4f78-8711-a4cc59e53d1e/hp_beverages_m_soups-680_250122_01.jpg" alt="" class="img-responsive category-img">
+                </div>
+            </div>
+        </div>
+        
+        <div class="spacer-8rem"></div>
+        <div class="spacer-8rem"></div>
+
```
Comment (swstica) [inline index.html]: Just a suggestion, this 8+8 rem gap looks too much btw the divs, you can reduce it.

**Review by swstica — APPROVED** [review]
Suggested a few changes, rest all looks good and can be merged.

File: css/homepage.css (hunk 1 above)
Comment (kaustubh-jsr) [reply css/homepage.css]: Thanks for the input. Done.

File: css/homepage.css (hunk 2 above)
Comment (kaustubh-jsr) [reply css/homepage.css]: Thanks for the suggestion. Done.

File: index.html (hunk 3 above)
Comment (kaustubh-jsr) [reply index.html]: Done

**Pull request merged**